api/src/main/java/org/cardanofoundation/rosetta/common/services/impl/CardanoServiceImpl.java

```diff
@@ -84,6 +80,7 @@ public class CardanoServiceImpl implements CardanoService {
 
   private final ProtocolParamService protocolParamService;
 
+  private final RestTemplate restTemplate;
```

Comment:
Don't want to nitpick, but this final field is better looking on the 82nd line since a block of final fields is shaping.